pkg/podvolume/backupper.go

```diff
@@ -312,6 +336,12 @@ func (b *backupper) BackupPodVolumes(backup *velerov1api.Backup, pod *corev1api.
 			continue
 		}
 		b.wg.Add(1)
+
+		if err := b.pvbIndexer.Add(volumeBackup); err != nil {
+			errs = append(errs, errors.Wrapf(err, "failed to create PodVolumeBackup for pvc %s", pvc.Spec.VolumeName))
```

Comment:
is it the write error message?

Shouldn't it be `failed to create indexer for pvb` and with the pvb information.  It seems there will be an error only when the input param is not a valid k8s object